### PR TITLE
Showcase Regwall: Localizes Google Sign-In button

### PR DESCRIPTION
**Pull request by ChrisAntaki**
The Google Sign-In button won't localize itself :shrug:

### CODE REVIEW

**Comment by ChrisAntaki** [comment]
image

**Comment by ChrisAntaki** [comment]
CC @ed-bird

File: src/utils/gaa.js [hunk 1]
```diff
@@ -259,7 +243,11 @@ body {
   display: none;
 }
 #${GOOGLE_SIGN_IN_BUTTON_ID} .abcRioButton.abcRioButtonBlue .abcRioButtonContents {
-  font-size: 15px !important;
+  font-size: 0 !important;
+}
+#${GOOGLE_SIGN_IN_BUTTON_ID} .abcRioButton.abcRioButtonBlue .abcRioButtonContents span[id^=not_signed_]:before {
```
Comment (elijahsoria) [inline src/utils/gaa.js:248]: Could you add a comment about what this span does and why you need this class and the one above it?

File: src/utils/gaa.js [hunk 2]
```diff
@@ -369,6 +357,16 @@ export class GaaMeteringRegwall {
    * @param {{ iframeUrl: string }} params
    */
   static render_({iframeUrl}) {
+    const languageCode = getLanguageCodeFromElement(self.document.body);
+
+    // Tell the iframe which language to render.
```
Comment (elijahsoria) [inline src/utils/gaa.js:362]: Prefer to use `addQueryParam` here: https://github.com/subscriptions-project/swg-js/blob/41f226737a782a2afd56d312ff7f093b804cdb9f/src/utils/url.js#L190

Comment (ChrisAntaki) [reply src/utils/gaa.js:362]: Good call, updated

File: src/utils/gaa.js (hunk 1 above)
Comment (ChrisAntaki) [reply src/utils/gaa.js:250]: I added a couple CSS comments, please let me know what you think

Comment (elijahsoria) [reply src/utils/gaa.js:250]: I think I get it now. If we have localized text, we set the default class font size to 0px and add this class with 15px font?

What most confuses me most is what `span[id^=not_signed_]` looks for and how the two classes work together. If you could add in some stuff about that in the comment that would be great.

**Review by elijahsoria — APPROVED** [review]

**Comment by ChrisAntaki** [comment]
> I think I get it now. If we have localized text, we set the default class font size to 0px and add this class with 15px font?

Just to clarify, the code in this PR is always hiding the default text and is then showing localized text instead, though it defaults to selecting "en" if nothing is specified

> What most confuses me most is what span[id^=not_signed_] looks for and how the two classes work together. If you could add in some stuff about that in the comment that would be great.

That's a good call out! I updated the code so both rules point to the "span[id^=not_signed_]" element. Does that simplify it to the point where the comments add enough clarity? Or would more clarity still be helpful?

**Comment by ChrisAntaki** [comment]
Basically, the first rule hides the button's default "Sign in with Google" text using `font-size`. Then the second rule creates a pseudo-element using [`::before`](https://developer.mozilla.org/en-US/docs/Web/CSS/::before) and gives it localized text content and a non-zero `font-size`

**Review by elijahsoria — APPROVED** [review]

**Pull request merged**